### DISCUSSION TITLE
请问可以禁止用户自己修改“专属标签”吗？

### DISCUSSION THREAD

**Discussion by netgc** [walinejs/waline#1607] [Q&A] [answered]
请问可以禁止用户自己修改“专属标签”吗？想仅在管理端修改。不知道是否有什么参数来配置？谢谢。

**Comment by lizheming** [comment, marked as answer]
暂不支持，欢迎 PR

**Reply by Mister-Hope** [reply]
其实这个问题加的时候我就想说了，我觉得支持用户自定义标签本来就是很奇怪的事情，因为本来注册的时候他们就可以自选昵称。

用户完全可以把自己的标签设置成管理员，甚至《管理员他爹》

**Reply by lizheming** [reply]
那昵称也是一样的，那不让用户设置昵称么。针对仿冒的问题是不是可以考虑差异化管理员标签样式呢。
